apps/opik-frontend/src/components/pages-shared/automations/AddEditRuleDialog/LLMJudgeRuleDetails.tsx

```diff
@@ -335,7 +335,7 @@ const LLMJudgeRuleDetails: React.FC<LLMJudgeRuleDetailsProps> = ({
                   <LLMPromptMessagesVariables
                     parsingError={parsingVariablesError}
                     validationErrors={validationErrors}
-                    projectId={form.watch("projectId")}
+                    projectId={form.watch("projectIds")[0] || ""}
```

Comment:
Same here, get an answer from Product before merging it

Reply:
Will check, how it is with FE -> BE name population